org.nanuc.Axolotl.yml

```diff
@@ -39,168 +29,74 @@ finish-args:
+        npm_config_cache: /run/build/axolotl/flatpak-node/npm-cache
+        # Sets the directory where Node is located so that npm won't download the headers.
+        npm_config_nodedir: /usr/lib/sdk/node18
+        # Cargo config
+        CARGO_HOME: /run/build/axolotl/cargo
+        # Cargo offline mode
+        CARGO_NET_OFFLINE: "true"
     build-commands:
-      - install -Dm 644 electron-linux-amd64-20.2.0.zip ${BUNDLER_CACHE}/electron-linux-amd64-20.2.0.zip
-    sources:
-      - type: file
-        url: https://github.com/electron/electron/releases/download/v20.2.0/electron-v20.2.0-linux-x64.zip
-        sha256: c0e3522de34819b838f4a35ddf30c6283c61be1bb8dff02089cda3f641938aad
-        dest-filename: electron-linux-amd64-20.2.0.zip
+      # Install the packages from our offline cache.
+      - npm --prefix=axolotl-web --offline install
 
-  - name: astilectron-bundler-electron-arm64
-    buildsystem: simple
-    only-arches:
-      - aarch64
-    build-options:
-      env:
-        BUNDLER_CACHE: /app/lib/astibundler/cache
-    build-commands:
-      - install -Dm 644 electron-linux-arm64-20.2.0.zip ${BUNDLER_CACHE}/electron-linux-arm64-20.2.0.zip
-    sources:
-      - type: file
-        url: https://github.com/electron/electron/releases/download/v20.2.0/electron-v20.2.0-linux-arm64.zip
-        sha256: 1d2dd05f060786eae5ce687794aa071df8a5f5733b7b2efa1481de8db75d7738
-        dest-filename: electron-linux-arm64-20.2.0.zip
+      # Build axolotl-web
+      - npm --prefix=axolotl-web --offline run build
 
-  - name: axolotl-electron-bundle
-    buildsystem: simple
-    build-options:
-      env:
-        npm_config_offline: "true"
-        npm_config_cache: /run/build/axolotl-electron-bundle/flatpak-node/npm-cache
-        GOBIN: /app/bin
-        GO111MODULE: off
-        GOPATH: /run/build/axolotl-electron-bundle
-    subdir: src/github.com/nanu-c/axolotl
-    build-commands:
-      - make build-dependencies-axolotl-web
-      - make build-axolotl-web
-      # Specify bundler working directory (where Electron and Astilectron is)
-      - echo "$(jq '. + { working_directory_path:"/app/lib/astibundler" }' bundler.json)" > bundler.json
-      - make build-axolotl-electron-bundle &
-        while [ -e /proc/$! ]; do echo -n "."; sleep 100; done
-      - make install-axolotl-electron-bundle DESTDIR=${FLATPAK_DEST} INSTALL_PREFIX=/bin
-    sources:
-      - generated-axolotl-web-sources.0.json
-      - generated-axolotl-sources.json
-      - type: git
-        url: https://github.com/nanu-c/axolotl
-        tag: v1.6.0
-        commit: e521ab32b5bf315866a17f6844e5339abbc41e79
-        dest: src/github.com/nanu-c/axolotl
-        x-checker-data:
-          type: git
-          tag-pattern: "^v([\\d.]+)$"
+      # Build axolotl
+      - cargo --offline fetch --manifest-path Cargo.toml --verbose
+      - cargo --offline build --features tauri --release --verbose
+      - install -Dm755 ./target/release/axolotl -t /app/bin/
```

Comment:
Lets stick with the same formatting?
eg a space between the letters and the numbers.

Also, lets use ${FLATPAK_DEST} rather than the hardcoded string? that is what I used across the Manifest already.

Lastly, lets also stick with the same folder style? eg. not ./target, but just target. Again, see the other parts of the manifest.

Reply:
Good idea. Done.